### PR TITLE
Redux toolkit

### PR DESCRIPTION
**Pull request by PuneetP16**
28k + changes in line of code is due to reformatting of the files (indentation and reshuffling). Just check the store folder for review

### CODE REVIEW

File: src/contexts/themeContext.js [hunk 1]
```diff
@@ -17,6 +20,80 @@ export const ThemeProvider = ({ children }) => {
...
+		// 	const { output } = await axiosRequest({
+		// 		method: "GET",
+		// 		url: playlistsURL,
+		// 		resKey: "playlists",
+		// 	});
+
+		// 	dispatchGet(videosAction.togglePlaylistFromPlaylists(output));
+		// })();
+
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (m26unkwn) [inline src/contexts/themeContext.js:94]: Remove this comment code.

**Review by saurabhchirde — APPROVED** [review]
Good to see that you have implemented Redux into your project. Couldn't find any issue in your code, You can proceed to merge.

File: src/pages/SingleVideo/SingleVideo.js [hunk 2]
```diff
@@ -105,94 +108,92 @@ export const SingleVideo = () => {
...
+										</button>
+										<button
+											className="btn btn--primary btn--icon btn--round playlist_icon"
+											title="Playlist"
+											onClick={togglePlaylistPanel}
+										>
+											{bxIcons.playlist}
+										</button>
+									</div>
+								</section>
```
Comment (m26unkwn) [inline src/pages/SingleVideo/SingleVideo.js:166]: you can break this component into smaller chunks.

**Review by m26unkwn — APPROVED** [review]
You can merge this code after doing few changes.

File: src/contexts/themeContext.js (hunk 1 above)
Comment (PuneetP16) [reply src/contexts/themeContext.js:94]: Kept it for development.

File: src/pages/SingleVideo/SingleVideo.js (hunk 2 above)
Comment (PuneetP16) [reply src/pages/SingleVideo/SingleVideo.js:166]: Will look into it.

**Pull request merged**